README.adoc

```diff
@@ -1,7 +1,7 @@
 = google-java-format-gradle-plugin
-:release-version: 0.8
-:default-google-java-format-version: 1.6
-:snapshot-version: 0.9-SNAPSHOT
+:release-version: 0.9
+:default-google-java-format-version: 1.7
+:snapshot-version: 0.10-SNAPSHOT
```

Comment:
This is only updated after a release/as part of a release commit. Please leave it untouched for now.

Reply:
Reverted this change